packages/react-storage/src/components/StorageBrowser/views/LocationDetailView/LocationDetailViewProvider.tsx

```diff
@@ -66,11 +66,13 @@ export function LocationDetailViewProvider({
     message: hasError ? message : downloadErrorMessage,
   });
 
+  const isNoActionAvailable = actions.every((obj) => obj.isHidden);
```

Comment:
Nit: could `obj` be a more descriptive name? Maybe just `action`?

Reply:
Yeah makes a sense to me, I'll try to use more descriptive names for variable going forward